oauthenticator/auth0.py

```diff
@@ -37,14 +37,28 @@
 from traitlets import Unicode
 
 from .oauth2 import OAuthenticator
+from .oauth2 import OAuthLogoutHandler
+
+
+class Auth0LogoutHandler(OAuthLogoutHandler):
+    async def render_logout_page(self):
+        if self.authenticator.logout_redirect_url:
+            self.redirect(self.authenticator.logout_redirect_url)
+            return
+
+        super().render_logout_page()
 
 
 class Auth0OAuthenticator(OAuthenticator):
 
     login_service = "Auth0"
 
+    logout_handler = Auth0LogoutHandler
+
     auth0_subdomain = Unicode(config=True)
 
+    logout_redirect_url = Unicode(help="""URL for logging out of Auth0""", config=True)
```

Comment:
Should this behavior be on the base class instead of just Auth0? Then the only thing in Auth0 is the default value for the url?

Reply:
That would make more sense! I just moved it! Thank you!